packages/nimble-components/src/spinner/styles.ts

```diff
@@ -144,27 +148,71 @@ export const styles = css`
         }
     }
 `.withBehaviors(
-    themeBehavior(
-        Theme.light,
+    appearanceBehavior(
+        SpinnerAppearance.default,
         css`
-            :host {
-                --ni-private-spinner-bits-background-color: ${Black91}
-        `
-    ),
-    themeBehavior(
-        Theme.dark,
-        css`
-            :host {
-                --ni-private-spinner-bits-background-color: ${Black15};
+            div.container {
+                opacity: 0.6;
             }
```

Comment:
Two things:
1. The default styles should be the default styles set on the control when no attributes are set. You don't need to use an appearanceBehavior for the default. In the top `div.container {}` you can set the default opacity style.
2. Check out the [responding to attribute appearance](https://github.com/ni/nimble/blob/main/packages/nimble-components/CONTRIBUTING.md#responding-to-attribute-values) guidelines. The key thing about behaviors is that thye should be used when "a large block of styles is overridden based on the attribute configuration". In this case we only are changing a single style so this approach seems a bit much.

I think you could organize a bit more like this:
```ts
export const styles = css`
    div.container { opacity: 0.6; }
    :host([appearance="accent"]) div.container { opacity: 1; }
`.withBehaviors(
    themeBehavior(Theme.light, css`
        :host {
            --ni-private-spinner-bits-background-color: ${Black91};
        }
        :host([appearance="accent"]) {
            --ni-private-spinner-bits-background-color: ${DigitalGreenLight};
        }
    `),
    ...
);
        
```